uaclient.conf

```diff
@@ -12,3 +12,5 @@ ua_config:
   apt_https_proxy: null
   http_proxy: null
   https_proxy: null
+  update_messaging_timer: null
```

Comment:
One idea that @mitechie proposed is to ship the config with the default values already in place.
If we do that, we can remove the constants we are using on the timer module

Reply:
Ack, but then, what happens if the user sets the value to `null`?
We would need to change some hereandtheres to make sure it will not surface the None value, as we would not have the constants.

So, imho, it is ok to put the defaults here, but keeping them in the file is safer.

Reply:
As discussed with @mitechie, we can define the same default values in the config file, but keep those in the constants as a fallback value for any invalid or missing value in the config file.